pkg/core/combat/gadget.go

```diff
@@ -17,6 +18,7 @@ var gadgetLimits []int
 func init() {
 	gadgetLimits = make([]int, EndGadgetTyp)
 	gadgetLimits[GadgetTypDendroCore] = 5
+	gadgetLimits[GadgetTypBountifulCore] = 5
```

Comment:
fyi, no need to change stuff: normal dendro cores and bountiful cores actually share the gadget limit and dmg icd but in sim you can't create a situation where dendro and bountiful cores exist at the same time

Reply:
actually, this is possible (a1 not up -> trigger bloom -> a1 up -> trigger bloom) so this needs to be fixed